network/forks/fork.go

```diff
@@ -56,14 +53,11 @@ func ForkDigestFromEpoch(currentEpoch types.Epoch, genesisValidatorsRoot []byte)
 // the active fork version in the node.
 func CreateForkDigest(
 	genesisTime time.Time,
-	genesisValidatorsRoot []byte,
+	genesisValidatorsRoot [32]byte,
 ) ([4]byte, error) {
 	if genesisTime.IsZero() {
 		return [4]byte{}, errors.New("genesis time is not set")
 	}
-	if len(genesisValidatorsRoot) == 0 {
-		return [4]byte{}, errors.New("genesis validators root is not set")
-	}
```

Comment:
This should rather be changed to comparison against zero hash, right?

Reply:
same applies here